src/screens/Settings.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import * as shape from 'd3-shape';
+import {Image, SafeAreaView, StyleSheet, View} from 'react-native';
+import {Block, Text} from '../components';
+import * as theme from '../constants/theme';
+import * as mocks from '../mocks';
+import {Line} from 'react-native-svg';
+import {LineChart, Path} from 'react-native-svg-charts';
+
+import {connect} from 'react-redux'; 
+
+interface OwnProps {
+    type: string;
+    user: any;
+    chart: Array<number>;
+    navigation:any;
+  }
+
+  type Props = OwnProps;
+
+export default class Settings extends React.Component<Props, {}> {
+    public static defaultProps = {
+        user: mocks.user,
+        chart: mocks.chart,
+      };
+    renderChart() {
```

Comment:
remove this function and everything that isn't just that text component saying this is settings